### PR TITLE
Fix Meeds Profile test cases

### PR DESCRIPTION
**Pull request by ichaieb**
Prior to this change , Meeds profile test cases were failed .After this change , they have been fixed .

### CODE REVIEW

File: src/test/java/io/meeds/qa/ui/stepDefinitions/UserProfileStepDefinition.java [hunk 1]
```diff
@@ -377,4 +373,14 @@ public void isProfileContactJobVisible(String job) {
     userProfileSteps.isProfileContactJobVisible(job);
   }
 
+  @Then("^The weekly point chart is displayed$")
```
Comment (boubaker) [inline src/test/java/io/meeds/qa/ui/stepDefinitions/UserProfileStepDefinition.java]: No need to regex when no method parameters

```suggestion
  @Then("The weekly point chart is displayed")
```

File: src/test/resources/features/Meeds/MeedsProfile.feature [hunk 2]
```diff
@@ -134,7 +135,7 @@ Feature: Search for User Informations in Profile page
     And I remove my profile work experiences 'QA Engineer'
     And I remove my profile work experiences 'IT Engineer Developer'
 
-  @ignore
+  @profile
```
Comment (boubaker) [inline src/test/resources/features/Meeds/MeedsProfile.feature]: Add `@profile` on top of the file once for all declared scenarios here

File: src/test/java/io/meeds/qa/ui/stepDefinitions/UserProfileStepDefinition.java (hunk 1 above)
Comment (ichaieb) [reply src/test/java/io/meeds/qa/ui/stepDefinitions/UserProfileStepDefinition.java]: done

File: src/test/resources/features/Meeds/MeedsProfile.feature (hunk 2 above)
Comment (ichaieb) [reply src/test/resources/features/Meeds/MeedsProfile.feature]: done

**Review by boubaker — APPROVED** [review]

**Pull request merged**